src/views/menus/navigation-menu.js

```diff
@@ -64,9 +64,8 @@ export class NavigationMenu extends LitElement {
         <li>No files loaded</li>
       `;
     } else {
-      files.forEach(file => {
-        filesList = html`
-          ${filesList}
+      let filesArray = files.map(file => {
+        return html`
```

Comment:
When you make a new PR, best to go from Master and not from another branch, otherwise you might end up with merging errors. I know ... I sometimes forget myself and have to jump through hoops :smile: